shared/create-new-app/templates/internal.js

```diff
@@ -0,0 +1,5 @@
+const main = async () => {
+  const websocket = createWebsocket();
```

Comment:
My first step when creating this is to add a `console.log('init internal app')` so that I can load them both and see the JS working. Is it worth adding this here?

Reply:
Where would you like the console.log? In the `main` function, or when the WebSocket connection is ready?

Reply:
In main I think, just as a very simple smoke-test that "yep, this code runs!"